README.adoc

```diff
@@ -112,27 +126,27 @@ include::finish/src/main/liberty/config/server.xml[tags=**]
 ---- 
 
 // =================================================================================================
-// Adding a health check to the system service
+// Adding health checks to the system service
 // =================================================================================================
 
-=== Adding a health check to the system service
+=== Adding health checks to the system service
 
 [role="code_command hotspot", subs="quotes"] 
 ---- 
-#Create the `SystemHealth` class.#
-`src/main/java/io/openliberty/guides/system/SystemHealth.java`
+#Create the `SystemReadinessCheck` class.#
+`src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java`
 ---- 
 
-SystemHealth.java
+SystemReadinessCheck.java
 [source, java, linenums, role='code_column']
 ----
-include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tags=SystemHealth]
+include::finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java[tags=SystemReadinessCheck]
 ----
 
-The [hotspot=8]`@Health` annotation indicates that this particular bean implements the 
-[hotspot=10]`HealthCheck` interface. By pairing this annotation with the [hotspot=9]`ApplicationScoped` 
-context from the Contexts and Dependency Injections API, the bean is discovered
-automatically when the http://localhost:9080/health[http://localhost:9080/health^] endpoint receives a request.
+The [hotspot=8 file=0]`@Readiness` annotation indicates that this is a readiness health check procedure. 
```

Comment:
Change
"The [hotspot=8 file=0]`@Readiness` annotation indicates that this is a readiness health check procedure."
To
"The [hotspot=8 file=0]`@Readiness` annotation indicates that this particular bean is a readiness health check procedure. "

**Just need a noun following "this" for reader's reference. (I assumed it was still "particular bean", as in previous text.)**